bin/improver-nbhood

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
Good point, I think so - this way is more like Rose and cylc and leaves us open to having improver subcommands in e.g. bash (for utilising tools like `nco`).